### ISSUE TITLE
Sharing feature is really hard to discover

### ISSUE THREAD

**Issue by jancborchardt** [penpot/penpot#855]
**Is your feature request related to a problem? Please describe.**
It took me quite a long time to realize how to share a file for others to view. I didn’t notice the little "View" icon on the top right and would have also not considered "Sharing" being in there.

**Describe the solution you'd like**
Since in the "View" mode there’s both "Share link" and "Edit page", the header in edit mode could also just have 2 buttons: "Share link" and "View page"?
![Share link not discoverable 3](https://user-images.githubusercontent.com/925062/115748611-9c70eb80-a396-11eb-8e27-9fdd307af564.png)
![Share link not discoverable 4](https://user-images.githubusercontent.com/925062/115748606-9bd85500-a396-11eb-8008-1b136329e148.png)

------------------

**Describe alternatives you've considered**
Other places I looked in for a "Share link" entry were the different menus:
![Share link not discoverable](https://user-images.githubusercontent.com/925062/115748592-9713a100-a396-11eb-8d13-c0bcb5277002.png)
![Share link not discoverable 2](https://user-images.githubusercontent.com/925062/115748586-967b0a80-a396-11eb-9c87-56bdb2af7b0c.png)

**Comment by elhombretecla** [comment]
Hi @jancborchardt Thank you for your feedback :)

We just create an enhancement in Taiga in order to achieve this as soon as we can.

You can rack the progress here [https://tree.taiga.io/project/penpot/issue/1604](https://tree.taiga.io/project/penpot/issue/1604)

**Comment by jdittrich** [comment]
I had a rather similar problem, [documented here](https://community.penpot.app/t/no-feature-request-finding-out-how-to-share-a-design/2245) but I never found that view mode. Even with @jancborchardt’s instructions I had trouble locating it. A colleage told me "there is no sharing" since they could not find it.

**Comment by pdurbin** [comment]
I just signed up for PenPot a few minutes ago. Yes, sharing is somewhat unintuitive, I'm afraid. 😢 

The thing that tripped me up is "No board found on the page". That is, from my file, I went to view mode (where it says "No boards found on the page" which seems strange) and then clicked "Share prototype". The dialog says "anyone with link will have access" (which is perfect because I don't want to require anyone to create a PenPot account) but when I copy the link into a different browser I see "No board found on the page".

After some Googling about what a PenPot board is, I drew one around my drawing and then it worked. I was able to share a public URL to my drawing/mockup/file/project/thing.

Overall, PenPot seems like a great tool! Thanks!

p.s. @jancborchardt @jdittrich oh hey, nice to see you here. 😄

**Comment by luizbills** [comment]
Penpot looks great but we need a feature to share the entire drawing area (not only the boards).

**Comment by jdittrich** [comment]
@luizbills I might not understand correctly – when you say "entire drawing area" do you mean sharing not only the content but also the possibility to interact and change content? Or do you mean not sharing boards one-by-one but the whole zoomed out overview?

**Comment by luizbills** [comment]
Yes. The whole zoomed out overview. The possibility of multiple users to
interact and change content will be a great feature.

**Comment by uglyeoin** [comment]
> @luizbills I might not understand correctly – when you say "entire drawing area" do you mean sharing not only the content but also the possibility to interact and change content? Or do you mean not sharing boards one-by-one but the whole zoomed out overview?

When you share a board... it is only that board.  If you need to navigate to another board, you need to share another link.  There needs to be an easy way to jump between boards on the same project.  For example I have a board for the admin section and for the front end section.  The client needs to sign off both boards.  

I struggle with sharing every time, which is how I ended up here, trying to work out how to do it.

My first thing was to look for sharing on the page.  Then I looked under "file".  

I expected the play button to do what it did, I just didn't expect the share button to be there.  I'm guessing you might want a "share prototype" for the client and a "share design" option for your team (albeit you can just share the link in that case).

**Comment by madalenapmelo-kp** [comment]
Hi jancborchardt,

The team has reviewed this issue and this is not something that we plan to address for the time being.  You can find more about the motivations for this decision at the Taiga issue: https://tree.taiga.io/project/penpot/issue/1604

**Comment by pdurbin** [comment]
> You can find more about the motivations for this decision at the Taiga issue: https://tree.taiga.io/project/penpot/issue/1604

The comment there is helpful:

"This was solved with the new sharing option from workspace. Also will be improved with this US: https://tree.taiga.io/project/penpot/us/11692?no-milestone=1 "
